tests/test_examples.py

```diff
@@ -484,6 +483,43 @@ def test_example_signing(tmp_path, request):
         _run_installer(input_path, installer, install_dir, request=request)
 
 
+@pytest.mark.skipif(sys.platform != "win32", reason="Windows only")
+@pytest.mark.skipif(
+        not shutil.which("azuresigntool") and not os.environ.get("AZURE_SIGNTOOL_PATH"),
+        reason="AzureSignTool not available"
+)
+@pytest.mark.parametrize(
+        "auth_method",
+        os.environ.get("AZURE_SIGNTOOL_TEST_AUTH_METHODS", "token,secret").split(","),
```

Comment:
Why do we need this "parametrized" parametrization? Are you using it locally while debugging? I don't see it in the CI setup 🤔

Reply:
I wanted to make sure that all authentication methods could potentially be covered. For tokens and secrets, I can just check the environment variables and skip if needed. For managed identities, however, this does not work because there isn't an environment variable to check for (logging into the vault must happen before AzureSignTool is executed).

So, the only "safe" methods to add into the parametrization are `token` and `secret`. However, I still wanted to allow testing with a managed identity.

Does that make sense? I tried to explain this briefly in the docstring.